src/nccl_ofi_rdma.cpp

```diff
@@ -136,15 +132,15 @@ static inline int free_base_req(uint64_t *num_inflight_reqs,
 static inline int check_post_rx_buff_req(nccl_net_ofi_rdma_req_t *rx_buff_req);
 
 
-static nccl_net_ofi_rdma_domain_t *rdma_endpoint_get_domain(nccl_net_ofi_rdma_ep_t *ep)
+nccl_net_ofi_rdma_domain_t *nccl_net_ofi_rdma_ep_t::rdma_endpoint_get_domain()
 {
-	return (nccl_net_ofi_rdma_domain_t*)ep->base.domain;
+	return (nccl_net_ofi_rdma_domain_t*)this->domain;
 }
 
 
-static nccl_net_ofi_rdma_device_t *rdma_endpoint_get_device(nccl_net_ofi_rdma_ep_t *ep)
+nccl_net_ofi_rdma_device_t *nccl_net_ofi_rdma_ep_t::rdma_endpoint_get_device()
 {
-	return (nccl_net_ofi_rdma_device_t*)rdma_endpoint_get_domain(ep)->base.device;
+	return (nccl_net_ofi_rdma_device_t*)this->rdma_endpoint_get_domain()->base.device;
 }
```

Comment:
I made these functions `inline` in their header declaration, I need to add the missing `inline` keywords in their definition here.

Reply:
Fixed in latest force push.